rts/lra/lra-filters/src/main/java/io/narayana/lra/filter/ClientLRAResponseFilter.java

```diff
@@ -36,16 +36,19 @@
     public void filter(ClientRequestContext requestContext, ClientResponseContext responseContext) throws IOException {
         Object incomingLRA = responseContext.getHeaders().getFirst(LRA_HTTP_HEADER);
 
-        if (incomingLRA == null) {
-            incomingLRA = requestContext.getProperty(LRA_HTTP_HEADER);
-        }
-
         /*
```

Comment:
Why this is removed ?

Reply:
It was a cut and paste error. BTW the test suite still passed so something is not right, I will put this PR on HOLD while I investigate further. Thanks